sample.env

```
@@ -0,0 +1,16 @@
+export TARGET_HOSTNAME=localhost
+export JMAP_TARGET_HOSTNAME=localhost
+export IMAP_SERVER_HOSTNAME=localhost
+export SMTP_TARGET_HOSTNAME=localhost
+export WEBADMIN_SERVER_HOSTNAMe=localhost
+export JMAP_PORT=443
+export JMAP_PROTOCOL=https
+export WEBADMIN_PORT=80
+export WS_PROTOCOL=wss
+export DURATION=15
+export IMAP_PORT=993
+export IMAP_PROTOCOL=imaps
+export USER_COUNT=1000
+export SMTP_PORT=25
+export SMTP_SSL_PORT=465
+export WEBADMIN_PROTOCOL=http
```

```suggestion
#export TARGET_HOSTNAME=localhost
#export JMAP_TARGET_HOSTNAME=localhost
#export IMAP_SERVER_HOSTNAME=localhost
#export SMTP_TARGET_HOSTNAME=localhost
#export WEBADMIN_SERVER_HOSTNAMe=localhost
#export JMAP_PORT=443
#export JMAP_PROTOCOL=https
#export WEBADMIN_PORT=80
#export WS_PROTOCOL=wss
#export DURATION=15
#export IMAP_PORT=993
#export IMAP_PROTOCOL=imaps
#export USER_COUNT=1000
#export SMTP_PORT=25
#export SMTP_SSL_PORT=465
#export WEBADMIN_PROTOCOL=http
```

Let's comment those by default, an admin can then uncomment as needed